classes/plugins/PubIdPlugin.inc.php

```
@@ -24,6 +24,67 @@ function __construct() {
+					if ($issueEnabled) {
+						$issueDao = DAORegistry::getDAO('IssueDAO');
+						$issues = $issueDao->getPublishedIssues($context->getId());
+						while ($issue = $issues->next()) {
+							$issuePubId = $issue->getStoredPubId($this->getPubIdType());
+							if (empty($issuePubId)) {
+								$issuePubId = $this->getPubId($issue);
+								$issueDao->changePubId($issue->getId(), $this->getPubIdType(), $issuePubId);
+							}
+						}
+					}
+					if ($submissionEnabled || $representationEnabled) {
+						$submissionDao = Application::getSubmissionDAO();
+						$representationDao = Application::getRepresentationDAO();
+						$publishedArticleDao = DAORegistry::getDAO('PublishedArticleDAO');
+						$publishedArticles = $publishedArticleDao->getPublishedArticlesByJournalId($context->getId());
+						while ($publishedArticle = $publishedArticles->next()) {
+							if ($submissionEnabled) {
+								$submissionPubId = $publishedArticle->getStoredPubId($this->getPubIdType());
+								if (empty($submissionPubId)) {
+									$submissionPubId = $this->getPubId($publishedArticle);
+									$submissionDao->changePubId($publishedArticle->getId(), $this->getPubIdType(), $submissionPubId);
+								}
+							}
+							if ($representationEnabled) {
+								$representations = $representationDao->getBySubmissionId($publishedArticle->getid(), $context->getId());
+								while ($representation = $representations->next()) {
+									$representationPubId = $representation->getStoredPubId($this->getPubIdType());
+									if (empty($representationPubId)) {
+										$representationPubId = $this->getPubId($representation);
+										$representationDao->changePubId($representation->getId(), $this->getPubIdType(), $representationPubId);
+									}
+								}
+							}
+						}
+					}
+					$notificationManager->createTrivialNotification($user->getId(), NOTIFICATION_TYPE_SUCCESS);
+				}
+				return new JSONMessage(true);
```

Should we return an error message of some kind if `$suffixGenerationStrategy == 'customId'`?